core/domain/skill_services.py

```diff
@@ -905,3 +905,16 @@ def skill_has_associated_questions(skill_id):
         question_models.QuestionSkillLinkModel.get_all_question_ids_linked_to_skill_id( # pylint: disable=line-too-long
             skill_id))
     return len(question_ids) > 0
+
+
+def get_skill_descriptions_by_skill_id(skill_id):
```

Comment:
Better to (a) inline this function, (b) do a get_multi instead of individual gets.

Reply:
Sorry, but what do you mean by inline this function?

Reply:
Drop having a separate function -- instead, just add the implementation directly at the point where this function is currently being called from.

Reply:
Done.